crime-evidence/src/main/java/uk/gov/justice/laa/crime/evidence/service/MaatCourtDataService.java

```diff
@@ -20,10 +20,10 @@ public class MaatCourtDataService {
     private final ServicesConfiguration configuration;
     private static final String RESPONSE_STRING = "Response from Court Data API: %s";
 
-    public Long getRepOrderCapitalByRepId(Integer repId) {
+    public Long getCapitalAssetCount(Integer repId) {
 
         ResponseEntity<Void> response = maatAPIClient.head(
-                configuration.getMaatApi().getRepOrderEndpoints().getRepOrderCapitalUrl(),
+                configuration.getMaatApi().getRepOrderEndpoints().getGetCapitalAssetCountUrl(),
```

Comment:
Fully aware that the getter here is auto-generated, but this isn't a great name. Perhaps we should instead rename the endpoint to `capitalAssetCountUrl` so that the getter contains only a single _get_?